native/src/components/MapView.tsx

```diff
@@ -59,12 +58,10 @@ const MapView = ({
   navigateTo,
   language,
   cityCode,
-  setUserLocation,
-  userLocation
+  onRequestLocationPermission,
+  locationPermissionGranted
 }: MapViewPropsType): ReactElement => {
-  const [followUserLocation, setFollowUserLocation] = useState<boolean>(false)
-  const [locationPermissionGranted, setLocationPermissionGranted] = useState<boolean>(false)
-
+  const [followUserLocation, setFollowUserLocation] = useState<boolean>(locationPermissionGranted && !selectedFeature)
```

Comment:
shouldn't we go to the bounding box if nothing is selected and keep the value false in the beginning?

Reply:
hm. we did never discuss this, actually. I think it is more intuitive to center around current position if enabled. But I also see why you would display the bounding box instead. @f1sh1918 what is your opinion here?

Reply:
i think its fine to use location. to gain a good overview we should keep the zoom level same as it would use the bounding box

Reply:
ok i'm fine with that to only use bbox if we have no user location

Reply:
here is the problem that its only set once and will not get true by changing ```locationPermissionGranted``` prop